<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="guitar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>121</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>吉他</string>
   </property>
  </widget>
  <widget class="QPushButton" name="piano1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>C2_钢琴</string>
   </property>
  </widget>
  <widget class="QPushButton" name="piano2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>190</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>C4_钢琴</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
